deployment/marketplace/digitialocean/one-click-droplet/scripts/90-cleanup.sh

```diff
@@ -22,6 +22,10 @@ rm -rf /var/lib/cloud/instances/*
 rm -f /root/.ssh/authorized_keys /etc/ssh/*key*
 touch /etc/ssh/revoked_keys
 chmod 600 /etc/ssh/revoked_keys
+sudo apt-get purge droplet-agent -y
```

Comment:
Do we need to remove it?
Seems like it will not provide statistics to DO if we will remove it.